src/components/scenes/TransactionListScene.tsx

```diff
@@ -187,6 +187,23 @@ function TransactionListComponent(props: Props) {
   // Renderers
   //
 
+  /**
+   * HACK: This `RefreshControl` doesn't actually do anything visually or
+   * functionally noticeable besides making Android scroll gestures actually
+   * work for the parent `Animated.FlatList`
+   */
+  const refreshControl = React.useMemo(() => {
+    return (
+      <RefreshControl
```

Comment:
We could add `Platform.os !== 'android' ? undefined : <RefreshControl... />` to keep things cleaner on iOS

Reply:
A bit of cognative load with the negative logic. I'll do `Platform.OS === 'ios' ? undefined` so that its
easier to read while retaining your preference for putting the null/undef case first.